tests/lib/wick-get-iface-ip.bats

```diff
@@ -86,5 +104,50 @@ setup() {
     local RESULT
 
     mock-command ifconfig wick-get-iface-ip/bsd
+    mock-command wickCommandExists wick-get-iface-ip/wick-command-exists-no-ip
     ! wickGetIfaceIp RESULT noip
 }
+
+@test "lib/wick-get-iface-ip: ip command - All interfaces" {
+    local result
+
+    mock-command ip wick-get-iface-ip/ip
```

Comment:
Is the `ip` commant not available on a mac?  Or does it have different output than Gnu/Linux?